### ISSUE TITLE
Autorest is failing in generating sdk with "[Exception] No input files provided."

### ISSUE THREAD

**Issue by 93mishra** [Azure/autorest#3953]
**Describe the bug**
```
Trying to generate sdk through autorest but getting error "[Exception] No input files provided." Even input file is configured in config file. 

(azcli) ➜  connectedvmware git:(master) autorest $azcli/azure-rest-api-specs-pr/specification/connectedvmware/resource-manager/readme.md --python --python-sdks-folder=$HOME/azcli/dev --python-mode=update
AutoRest code generation utility [cli version: 3.1.1; node: v14.15.5, max-memory: 4096 MB]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest

There is a new version of AutoRest available (3.1.2).
 > You can install the newer version with with npm install -g autorest@latest

NOTE: AutoRest core version selected from configuration: ~3.0.6298.
   Loading AutoRest core      '/home/santosh/.autorest/@autorest_core@3.0.6375/node_modules/@autorest/core/dist' (3.0.6375)
   Loading AutoRest extension '@autorest/python' (latest->5.6.4)
   Loading AutoRest extension '@autorest/modelerfour' (^4.15.456->4.17.1)
**[Exception] No input files provided.**



- OpenAPI files having the issues
- Autorest command used
```
autorest $azcli/azure-rest-api-specs-pr/specification/connectedvmware/resource-manager/readme.md --python --python-sdks-folder=$HOME/azcli/dev --python-mode=update

**Expected behavior**
SDK should get generated because on previous version of autorest same cmd able to generate sdk. 

**Additional context**
Running cmd on linux machine.

**Comment by 93mishra** [comment]
Attaching readme.md (config) file 
[readme.txt](https://github.com/Azure/autorest/files/6090862/readme.txt)

**Comment by timotheeguerin** [comment]
This issue is a duplicate of #3895 which has been fixed in core version `3.1.0`. Please update to that or revert to version `3.0.6371`
